db/migrate/20160526142456_add_payment_mode_to_tickets.rb

```diff
@@ -0,0 +1,5 @@
+class AddPaymentModeToTickets < ActiveRecord::Migration
+  def change
+    add_column :tickets, :payment_mode, :string
```

Comment:
yes... i will edit it keeping "Offline" as default payment_mode as the existing tickets required offline payments